pages/dkp/kommander/2.2/networking/use-istio/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: layout.pug
+navigationTitle: Integrate microservices using Istio
+title: Integrate microservices using Istio
+menuWeight: 24
+excerpt: Learn how to integrate microservices managed by Istio into a DKP cluster
+beta: false
+experimental: true
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+Istio helps you manage cloud-based deployments by providing an open-source service mesh to connect, secure, control, and observe microservices.
+
+This tutorial demonstrates how to expose an application running on the DKP cluster using the LoadBalancer (layer-4) service type.
+
+<p class="message--note"><strong>NOTE: </strong>Usage and installation of Istio on DKP is currently a self-service feature.</p>
+
+## Before you begin
+Before starting this tutorial, you should verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster. For information about deploying Kubernetes with default settings, see the [Quick start][quickstart].
```

Comment:
Quickstart does not exist anymore so this link is broken.

Reply:
Removed second sentence.